Engine/source/math/mPoint2.h

```diff
@@ -285,6 +288,13 @@ inline Point2I::Point2I(S32 _x, S32 _y)
 }
 
 
+inline Point2I::Point2I(S32 xy)
+   : x(xy), y(xy)
+{
+   //
```

Comment:
Yeah cleaner without them but just matching what is already there ;)  https://github.com/rextimmy/Torque3D/blob/mPointX_constructors/Engine/source/math/mPoint2.h#L271-L288